app/controllers/tag_controller.rb

```diff
@@ -11,6 +11,10 @@ def index
 
     @title = I18n.t('tag_controller.tags')
     @paginated = true
+    @order_type = params[:order] == "desc" ? "asc" : "desc"
```

Comment:
I'm a little bit confused... if the `order` param is `desc`, we want to set the `@order_type` to `desc`...?

Reply:
@shauncarland The logic behind this is that, we have to invert the order when a user click on sorting header again. So lets say if current shown order is according to "name descending" then we want front end to request on url with order type be ascending (opposite of descending). I hope you got my point :smile:

Reply:
🎉 🎉 🎉 🎉 🎉 🎉